tests/e2e/transaction/generate_transaction.test.ts

```diff
@@ -0,0 +1,197 @@
+import { AptosConfig, Network, Aptos, Account, Deserializer, U64 } from "../../../src";
+import {
+  RawTransaction,
+  TransactionPayloadScript,
+  TransactionPayloadMultisig,
+  TransactionPayloadEntryFunction,
+} from "../../../src/transactions/instances";
+import { longTestTimeout } from "../../unit/helper";
+import { fundAccounts, singleSignerScriptBytecode } from "./helper";
+
+describe("generate transaction", () => {
+  const config = new AptosConfig({ network: Network.LOCAL });
+  const aptos = new Aptos(config);
+  const senderAccount = Account.generate();
+  const recieverAccounts = [Account.generate(), Account.generate()];
+  const secondarySignerAccount = Account.generate();
+  const feePayerAccount = Account.generate();
```

Comment:
I wonder, should we do some test against different scheme type of account?

For example:
senderAccount = Ed25519
recieverAccount = Secp256k1
...

Reply:
yes. we can add it on simulation/submission tests, generate transaction doesnt really care for the signing scheme